static/app/views/organizationGroupDetails/index.tsx

```diff
@@ -14,34 +14,18 @@ type Props = {
   isGlobalSelectionReady: boolean;
   organization: Organization;
   projects: Project[];
+  loadingProjects: boolean;
```

Comment:
is this prop being used?

Reply:
Nope, it's not. Thanks for catching this

(I was experimenting with something, which is how this ended up here)